components/match2/wikis/brawlhalla/match_summary.lua

```diff
@@ -20,7 +20,7 @@ local Opponent = OpponentLibraries.Opponent
 
 local GREEN_CHECK = Icon.makeIcon{iconName = 'winner', color = 'forest-green-text', size = 'initial'}
 local DRAW_LINE = Icon.makeIcon{iconName = 'draw', color = 'bright-sun-text', size = 'initial'}
-local NO_CHECK = '[[File:NoCheck.png|link=]]'
+local NO_CHECK = '[[File:NoCheck.png|link=|16px]]'
```

Comment:
maybe use Module:Icon here?

Reply:
It's a filler icon. The preferred outcome would that it's not be used at all, but currently is to easier align things